server/commands/QueueNextCommand.ts

```diff
@@ -0,0 +1,46 @@
+import CommandDefinition from '../domain/CommandDefinition';
+import IoConnection from '../clients/IoConnection';
+import Song from '../domain/Song';
+import SongRepository from '../repositories/SongRepository';
+import QueueEventMessage from '../domain/event-messages/QueueEventMessage';
+import FetchVideoTitle from '../helpers/FetchVideoTitle';
+import SongService from '../services/SongService';
+
+async function createSongFromId(youtubeId: string) : Promise<Song> {
+  const name = await FetchVideoTitle.fetch(youtubeId);
+
+  const song: Song = {
+    name,
+    youtubeId,
+    trimStartSeconds: null,
+    trimEndSeconds: null,
+    timesPlayed: 0,
+  };
+
+  return song;
+}
+
+async function queueNext(songName: string) : Promise<string> {
+  const song: Song = await SongRepository.getByName(songName);
+  const potentialYoutubeId = songName.split(' ')[0];
+  const actualSong = song
```

Comment:
This could be simplified to simple or statement I think?
```suggestion
  const actualSong = song || await createSongFromId(potentialYoutubeId);
```